### PR TITLE
update for 17.0

### PR DESCRIPTION
**Pull request by OnGle**

### CODE REVIEW

File: conf.d/main [hunk 1]
```diff
@@ -1,29 +1,15 @@
-#!/bin/sh -ex
-
-DB_NAME=oscommerce
-DB_USER=oscommerce
+#!/bin/bash -ex
+DB_NAME=oscommerce DB_USER=oscommerce
```
Comment (JedMeister) [inline conf.d/main]: Any reason why you moved these to be on the same line?

Functionally it should work fine, but I'm curious why you did it?

File: conf.d/main [hunk 2]
```diff
@@ -1,29 +1,15 @@
...
 DB_PASS=$(mcookie)
 
 ADMIN_NAME=admin
-ADMIN_PASS=turnkey
-ADMIN_MAIL=admin@example.com
+ADMIN_PASS=turnkey ADMIN_MAIL=admin@example.com
```
Comment (JedMeister) [inline conf.d/main]: And again - Any reason why you moved these to be on the same line?

File: conf.d/main [hunk 3]
```diff
@@ -43,37 +32,81 @@ MYSQL_ADMIN="mysqladmin --user=root --password=$MYSQL_PASS"
...
-INS_USR="CFG_STORE_NAME=TurnKey+osCommerce&CFG_STORE_OWNER_NAME=Admin&CFG_STORE_OWNER_EMAIL_ADDRESS=$EMAIL&CFG_ADMINISTRATOR_USERNAME=$ADMIN_NAME&CFG_ADMINISTRATOR_PASSWORD=$ADMIN_PASS&x=29&y=15"
-
-$CURL "${URL}"
-$CURL "${RPC}?action=dbCheck&${RPC_DB}"
-$CURL "${RPC}?action=dbImport&${RPC_DB}"
-
-$CURL "${URL}?step=2" --data "${INS_DB}"
-$CURL "${URL}?step=3" --data "${INS_DB}&${INS_WWW}"
-$CURL "${URL}?step=4" --data "${INS_DB}&${INS_WWW}&${INS_USR}"
+CURL="curl --insecure -L -c /tmp/cookie -b /tmp/cookie"
```
Comment (JedMeister) [inline conf.d/main:43]: Wow man - this was a HUGE update! Looks like lots of trial and error work here! Good work.

**Review by JedMeister — COMMENTED** [review]
Look great man. And looks like it was tons of work! Good job mate! :smile: 

All looks good, although I did have a question (in my review).</p>

File: conf.d/main (hunk 1 above)
Comment (OnGle) [reply conf.d/main]: My guess is pressing `j` with caps on in vim :p Good catch, I'll sort this.

File: conf.d/main (hunk 2 above)
Comment (OnGle) [reply conf.d/main]: (Same as above)

File: conf.d/main (hunk 3 above)
Comment (OnGle) [reply conf.d/main:43]: Cheers, yeah it was a fair bit of screwing about but got it working in the end :+1:

**Review by Christie018 — APPROVED** [review]
The changes to update the oscommerce appliance looks good to me. My only comment would be to maybe update the changelog with more code related changes done for the appliance for future reference.

**Pull request merged**